src/stream/src/executor/over_window/over_partition.rs

```diff
@@ -760,577 +928,3 @@ impl<'a, S: StateStore> OverPartition<'a, S> {
-                    cache.insert(
-                        CacheKey::Normal(
-                            StateKey {
-                                // order key doesn't matter here
-                                order_key: vec![].into(),
-                                pk: OwnedRow::new(vec![Some($pk.into())]).into(),
-                            },
-                        ),
-                        // value row doesn't matter here
-                        OwnedRow::empty(),
-                    );
-                )*
-                cache
-            }
-        };
-    }
-
-    macro_rules! create_change {
-        (Delete) => {
-            Change::Delete
-        };
-        (Insert) => {
-            Change::Insert(OwnedRow::empty())
-        };
-    }
-
-    macro_rules! create_delta {
-        ($(( $pk:literal, $change:ident )),* $(,)?) => {
-            {
-                #[allow(unused_mut)]
-                let mut delta = BTreeMap::new();
-                $(
-                    delta.insert(
-                        CacheKey::Normal(
-                            StateKey {
-                                // order key doesn't matter here
-                                order_key: vec![].into(),
-                                pk: OwnedRow::new(vec![Some($pk.into())]).into(),
-                            },
-                        ),
-                        // value row doesn't matter here
-                        create_change!( $change ),
-                    );
-                )*
-                delta
-            }
-        };
-    }
-
-    fn assert_ranges_eq(
-        result: Vec<(&CacheKey, &CacheKey, &CacheKey, &CacheKey)>,
-        expected: impl IntoIterator<Item = (ScalarImpl, ScalarImpl, ScalarImpl, ScalarImpl)>,
-    ) {
-        result
-            .into_iter()
-            .zip_eq(expected)
-            .for_each(|(result, expected)| {
-                assert_eq!(
-                    result.0.as_normal_expect().pk.0,
-                    OwnedRow::new(vec![Some(expected.0)])
-                );
-                assert_eq!(
-                    result.1.as_normal_expect().pk.0,
-                    OwnedRow::new(vec![Some(expected.1)])
-                );
-                assert_eq!(
-                    result.2.as_normal_expect().pk.0,
-                    OwnedRow::new(vec![Some(expected.2)])
-                );
-                assert_eq!(
-                    result.3.as_normal_expect().pk.0,
-                    OwnedRow::new(vec![Some(expected.3)])
-                );
-            })
-    }
-
-    #[test]
-    fn test_insert_delta_only() {
-        let cache = create_cache!();
-        let delta = create_delta!((1, Insert), (2, Insert), (3, Insert));
-        let calls = vec![create_call(Frame::rows(
-            FrameBound::Preceding(2),
-            FrameBound::Preceding(1),
-        ))];
-        let affected_ranges = find_affected_ranges(&calls, DeltaBTreeMap::new(&cache, &delta));
```

Comment:
Where did we migrate these tests to

Reply:
These are migrated to `frame_finder.rs`. The original tests for `find_affected_ranges` are removed. Because most of the logic of the original `find_affected_ranges` is moved to `frame_finder.rs`, I guess it can be enough to only test those `find_xxx_for_rows_frames` functions.